src/ios/registerNativeModule.js

```diff
@@ -50,6 +54,73 @@ const addProjectToLibraries = (libraries, file) => {
 };
 
 /**
+ * Given folder, it returns an array of all header files
+ * inside it, ignoring node_modules and examples
+ */
+const getHeadersInFolder = (folder) =>
+  glob
+    .sync('**/*.h', {
+      cwd: folder,
+      nodir: true,
+      ignore: GLOB_EXCLUDE_PATTERN,
+    })
+    .map(file => path.join(folder, file));
```

Comment:
we map to absolute path so we don't have to pass `destinationFolder` to `getHeaderSearchPath` because it's already a part of `header` path. (see method below)